device/mellanox/x86_64-nvidia_sn2201-r0/ACS-SN2201/sai_2201.xml

```diff
@@ -1,21 +1,4 @@
 <?xml version="1.0"?>
-<!--
```

Comment:
should we still keep the copyright info? looks like a common practice to have copyright in master

Reply:
fixed